tests/cloud/conftest.py

```diff
@@ -58,15 +59,17 @@ def __init__(self,model_name,num_cores,prompt,prompts_txt_file_path,aic_enable_d
         self.batch_size = batch_size
         self.prompt_len = prompt_len
         self.ctx_len = ctx_len
+        self.generation_len = None
         self.mxfp6 = mxfp6
         self.mxint8 = mxint8
+        self.full_batch_size = None

```

Comment:
Can you enable one test with full_batch_size, current supported models with this flags are llama, mistral, mixtral.
So for the tiny llama card you can pass full_batch_size and get the test verified.

Reply:
Doing it now.